enterprise_gateway/itests/test_authorization.py

```diff
@@ -20,23 +20,64 @@ def setUp(self):
     def tearDown(self):
         pass
 
-    def test_authorized_users(self):
+    def test_authorized_users_when_all_values_are_passed(self):
```

Comment:
Can we please do something about these _when clauses_ embedded in the test names?

What are the _values_ in _when all values are passed_?  

How about `test_complete_authorized_users`?